Marlin/temperature.cpp

```diff
@@ -85,10 +89,16 @@
 #endif
 #define PID_dT ((OVERSAMPLENR * 12.0)/(F_CPU / 64.0 / 256.0))
 
-int target_temperature[EXTRUDERS] = { 0 };
+#ifndef SINGLENOZZLE
+  int target_temperature[EXTRUDERS] = { 0 };
+  int current_temperature_raw[EXTRUDERS] = { 0 };
+  float current_temperature[EXTRUDERS] = { 0.0 };
+#else
+  int current_temperature_raw[1] = { 0 };
```

Comment:
Replacing EXTRUDERS with HOTEND.

Make this everywhere you normally needs temperature or stuff you need for a HOTEND.

And then:
```
#ifndef SINGLENOZZLE
  #define HOTEND EXTRUDER
#else
  #define HOTEND 1
#endif
```